executor/aggregate.go

```diff
@@ -45,39 +105,490 @@ type HashAggExec struct {
 	GroupByItems  []expression.Expression
 	groupKey      []byte
 	groupVals     [][]byte
+
+	// After we support parallel execution for aggregation functions with distinct,
+	// we can remove this attribute.
+	doesUnparallelExec bool
```

Comment:
unparallelExec is enough.

Reply:
`unparallelExec` will be duplicated with the member function name of HashAggExec.

Reply:
is `isUnparallelExec`  better?